src/SFA.DAS.LevyTransferMatching.Web/Orchestrators/ApplicationsOrchestrator.cs

```diff
@@ -110,22 +110,25 @@ public async Task<ApplicationViewModel> GetApplication(ApplicationRequest reques
             };
         }
 
-        public async Task AcceptFunding(AcceptFundingPostRequest request, CancellationToken cancellationToken = default)
+        public async Task SetApplicationAcceptance(ApplicationPostRequest request)
         {
-            var application = await _applicationsService.GetApplication(request.AccountId, request.ApplicationId, cancellationToken);
+            var application = await _applicationsService.GetApplication(request.AccountId, request.ApplicationId);
```

Comment:
I think this call is unnecessary since we're checking if the application is null in the inner api.

Reply:
Good point; removed.